Update Front-end/Pruebas/INDEX - FernandoYañez.html

NandoYanez opened the pull request.

NandoYanez committed:
Update Front-end/Pruebas/INDEX - FernandoYañez.html

Commit:
Diagrama Modular

Commit:
Ejemplo de Mapa de chile con Amcharts

este es un ejemplo de Mapa de chile con la extensión de javascript Amcharts

FernandoAedo committed:
commit

aquí subirán lo investigado

FernandoAedo committed:
commit

presentación para martes 16/10/18

FernandoAedo committed:
commit

me falto agregar un diagrama

FernandoAedo committed:
commit

diagrama no necesario eliminado

Commit:
Diagrama Uml

Commit:
Presentacion Mostrada 16-10-2018

Commit:
Prueba con leafletjs

Se crea un mapa , con un marcador , el mapa esta con las coordenadas de temuco .

Commit:
Mejora de Mapa con Libreria Ammaps

en este 2° subida , se modifica el codigo de prueba del mapa de chile , se agrega el codigo para cambiar el color dependiendo la region , donde son 10 posibles colores 1 seria lo mas claro , donde hay menos y 10 seria mas oscuro , donde el valor es mayor.

MarceloCastillo98 committed:
Add files via upload

jcredberry reviewed (commented):
Recuerden que no vamos a usar plotly sino D3.js.

Commit:
Merge remote-tracking branch 'origin/desarrollo' into desarrollo

Commit:
Pruebas de toma de datos json

Commit:
Prueba Con Jquery

Se realizan pruebas y repaso de jquery , como se muestra en la pagina subida , botones que realizan acciones simples de jquery , como movimiento entre otras

MarceloCastillo98 committed:
Add files via upload

MarceloCastillo98 committed:
Add files via upload

juan-cid committed:
prueba css de @font-face

Se buscaron posibles fuentes(fonts) para mejorar la visual de los textos(párrafos) y títulos de la pagina web.

NandoYanez committed:
Add files via upload

Commit:
Pruebas de Jquery con Css

En esta prueba , se busca comprender como se realiza lo de mostrar las imágenes con datos en la pagina de vote view , intentando realizar una copia o entender de como realizan esa función , de mostrar todos los que votan , y con poner el mouse sobre sus nombres aparezcan datos y fotos de la persona

Commit:
Merge remote-tracking branch 'origin/desarrollo' into desarrollo

MarceloCastillo98 committed:
Add files via upload

MarceloCastillo98 committed:
Add files via upload

MarceloCastillo98 committed:
Add files via upload

FernandoAedo committed:
Corrección gráfico UML

grafico nuevo

FernandoAedo committed:
Carta Gantt (1)

falta distribuir horas aun

FernandoAedo committed:
Corrección Gráfico integración de sistemas

corrección grafico

csanhueza commented:
Bien chicos a seguir así, ésta semana ya hay que tener, vistas previas y comienzo del maqueteo

NandoYanez committed:
prueba

NandoYanez committed:
prueba2

NandoYanez committed:
prueba borrada

FernandoAedo committed:
diagrama modular

diagrama modular version 1

NandoYanez committed:
Merge remote-tracking branch 'origin/desarrollo' into desarrollo

MarceloCastillo98 committed:
diagrame modular

Commit:
Maqueteo de Prueba para index 

en esta subida se realiza la creación de las carpetas y el maqueteo de html  del index de la pagina

Commit:
Modifiacion de Maqueteo

en esta subida se agrega un slide con imágenes , donde se cambia automáticamente , cada 5 segundos

Commit:
Mejoras index

Se cambio el css , por Boostrap para tener una pagina responsive , se agregaron algunas cosas , como el menú de navegación principal , entre otras cosas

NandoYanez committed:
Prueba de pagina

Prueba de pagina colocando un gráfico en el lugar deben ir cuando esté finalizada la pagina.

Commit:
Merge remote-tracking branch 'origin/desarrollo' into desarrollo

NandoYanez committed:
Prueba de pagina 2 - Graficos

MarceloCastillo98 committed:
prueba de mapa

MarceloCastillo98 committed:
mapa de chile

este archivo contiene un geojson con todo el sector continental de chile pero en baja resolucion

juan-cid committed:
Mejoras css

juan-cid committed:
css index 28/10

NandoYanez committed:
Prueba de Pagina 3 - Mapa de ubicación

Se agrega Mapa con ubicación Campus norte, todo esto en la pestaña "Contacto".

NandoYanez committed:
Prueba de pagina 3 - Mapa de ubicación - Correcion

Correccion en la barra de navegación, al presionar en Contacto no se podía volver a inicio presionando en la pestaña

NandoYanez committed:
Prueba de pagina  4

Nuevo css

Commit:
Añadido Buscardor en Nav

Commit:
Merge remote-tracking branch 'origin/desarrollo' into desarrollo

Commit:
Cambio de color  de ventana activa en Nav

FernandoAedo committed:
enlace para excel 

enlace

MarceloCastillo98 committed:
mapa de chile

mapa de chile con geojson

juan-cid committed:
prueba mapa chile

MarceloCastillo98 committed:
arreglo de posición mapa

Commit:
Mejoras en Pagina Prueba

se Cambio el slideshow , por uno con Boostrap , también se añadió en el navegador la barra extensible de lista

Commit:
Prueba de Open data xml

FernandoAedo committed:
Diagrama

Commit:
Merge remote-tracking branch 'origin/desarrollo' into desarrollo

juan-cid committed:
Mejoras pagina prueba

se agrego nueva pagina contenido.html y pruebas de mapas y css

juan-cid committed:
css de contenido.html

Commit:
Añadido Html de Contacto 

Se agrega html de contacto , en el cual contiene el mapa y formulario para enviar mensajes

Commit:
Merge remote-tracking branch 'origin/desarrollo' into desarrollo

Commit:
Cambiado Nombre html contacto

MarceloCastillo98 committed:
mapa con mas funciones

Commit:
Corregido Error de Contacto.html

se quito el error , que no aparecia en pantalla el mapa

Commit:
Merge remote-tracking branch 'origin/desarrollo' into desarrollo

Commit:
Creado Sistema de búsquedas Basico(XAMPP)

se crea un sistema de busqueda de diputados , muy basico , en donde se ingresa el nombre de un diputado y busca en un xml , si encuentra el dato , lo agrega al html

MarceloCastillo98 committed:
Add files via upload

juan-cid committed:
maquetas de paginas realizadas con mockflow.com

FernandoAedo committed:
carpeta para Sprint1

...

FernandoAedo committed:
maqueteo

maqueteo grafico_mapa

FernandoAedo committed:
sprint 1

maqueteo para sprint1

FernandoAedo committed:
Presentacio

agregar excel

Commit:
Cambiada Las imagenes  del Slider

Commit:
Cambiada clase de section index

FernandoAedo committed:
power sprint1

power sprint1

Commit:
Mejora de Resposive 

Se mejora el responsive , se hace unión entre las paginas , también se crea el maqueteo de la parte de votaciones , entre otros

MarceloCastillo98 committed:
ejemplo de buscador de cuidades 

lee un json en linea

Commit:
Pruebas Con jquery y Json

Commit:
Merge remote-tracking branch 'origin/desarrollo' into desarrollo

Commit:
Json para crear el Grafico ( Datos de Prueba )

Commit:
Pruebas para crear etiquetas automaticas con json

En este commit , se subio un codigo con js , para crear automaticamente etiquetas cuando se recorre el json

Commit:
Prueba de Mapa ojo de pescado 

se busco un codigo como ejemplo para empezar a crear el mapa que se solicito .

juan-cid committed:
modificación, mejoras css

Commit:
Agregado diputados (Corregir Error de Footer)

NandoYanez committed:
Arreglo de margenes

Arreglo de margenes de la pagina de prueba, en el Section

NandoYanez committed:
Merge remote-tracking branch 'origin/desarrollo' into desarrollo

Commit:
Arreglado Margenes  v2

Commit:
Creado Mapa con Datos  json , Mapa de dispersión

Commit:
Mejorado Mapa Dispersion

juan-cid committed:
Mejoras css

FernandoAedo committed:
contenido para Sprint 2

!leer el txt!

FernandoAedo committed:
menus

esta implementado el menu superior y menu lateral

Commit:
Modificacion a Pagina Nueva

Commit:
Modificacion a js que muestra xml

Commit:
Cambiado Contacto

MarceloCastillo98 committed:
buscador mejorado

FernandoAedo committed:
asd

asd

MarceloCastillo98 committed:
Merge remote-tracking branch 'origin/desarrollo' into desarrollo

Commit:
Presentacion sprint 2

MarceloCastillo98 committed:
Merge remote-tracking branch 'origin/desarrollo' into desarrollo

juan-cid committed:
Mejora presentación

Commit:
Ejemplo Mapa 2 de Gráfico scatterplot

Commit:
Agregada mejoras a pagina 

Creacion de perfil diputados , con conexión a base de datos mongodb, ejemplos de como conectarse a mongodb con php , cambiado diputados

Commit:
Agregado Datos en index

juan-cid committed:
agrega Acercade.html

MarceloCastillo98 committed:
Merge remote-tracking branch 'origin/desarrollo' into desarrollo

Commit:
Agregado buscador por ID o Materia .

MarceloCastillo98 committed:
Merge remote-tracking branch 'origin/desarrollo' into desarrollo

MarceloCastillo98 committed:
buscador

Commit:
Mejora de Grafico , filtrar partidos

Commit:
Añadido redireccionamiento a  proyectos

Commit:
Mejora Boletin

Muestra todos los diputados que votaron en x proyecto .

juan-cid committed:
Merge remote-tracking branch 'origin/desarrollo' into desarrollo

juan-cid committed:
mejoras de la pagina

Commit:
Ejemplo de menu popup para tabla diputados

Commit:
Añadido ventana de datos diputados en Descripcion.php

Commit:
Terminado ventana de datos diputados

MarceloCastillo98 committed:
Merge remote-tracking branch 'origin/desarrollo' into desarrollo

Commit:
Añadido funciones de buscador y seleccion multiple

contiene el codigo para detectar que opción se eligió y avisar cuando se presionan 2

juan-cid committed:
mejoras

Commit:
Buscador de Diputados agregado

Commit:
FINAL

csanhueza committed:
Actualización
